distributions/JavaScript/INSTRUCTIONS

```diff
@@ -0,0 +1,27 @@
+1. Install Enscriptem
+    # Get the emsdk repo
+    git clone https://github.com/emscripten-core/emsdk.git
```

Comment:
Is it a good idea to clone emscripten to current directory ? In such way we will have one GIT repo inside another. Be more specific on where to clone.

Does it also make sense to download repo with all its history ? For installation we only need last commit. It's a waste of disk space and bandwith.

Reply:
I agree, this will be handled in a different way now, there's a script to install emsdk to library repo's parent directory, in a emsdk subfolder by default and this can be overridden with a parameter.